lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -480,7 +485,23 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
                       <>
                         <div className="sm:col-span-2 border-b" />
                         <PanelLabel isCurrentStep verticalAlign="center">
-                          Max points
+                          <div className="flex items-center justify-end">
+                            Max points
+                            <IconButton
+                              icon={InfoIcon}
+                              title="About max points"
+                              onClick={() =>
+                                setMaxPointsPopoverOpen(prev => !prev)
```

Comment:
```suggestion
                                setMaxPointsPopoverOpen(open => !open)
```

Make it clear what `prev` means here.

Reply:
Muscle memory 😅